toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1893,7 +1896,7 @@ func ProvisionUserSSHCerts(installChroot safechroot.ChrootInterface, username st
 	allSSHKeys = append(allSSHKeys, sshPubKeys...)
 
 	for _, pubKey := range allSSHKeys {
-		logger.Log.Infof("Adding ssh key (%s) to user (%s) .ssh/authorized_users", filepath.Base(pubKey), username)
+		logger.Log.Infof("Adding ssh key (%s) to user (%s)", filepath.Base(pubKey), username)
```

Comment:
nit: why not display the full path?

Reply:
This is an INFO log. I felt that adding the path was a little too noisy for INFO. Hence, I removed it.